src/components/views/rooms/RoomListPanel/RoomListItemView.tsx

```diff
@@ -141,7 +146,7 @@ export const RoomListItemView = memo(function RoomListItemView({
         </Flex>
     );
 
-    if (!vm.showContextMenu) return content;
+    if (!vm.showContextMenu || listIsScrolling) return content;
```

Comment:
A comment will be great to explain why we don't want to display the context menu when scrolling